### ISSUE TITLE
Magento 2.1.3 - Syntax Error while Newsletter Sending

### ISSUE THREAD

**Issue by shahankitb997** [magento/magento2#8085]
### Preconditions

1.  Using Magento 2 CE Version 2.1.3
2.  Go to Admin -> Marketing -> Newsletter Template -> Add New Template/Edit Existing Email Template

### Steps to reproduce

1. In Sender Name Put with SPACE. If you don't put space then it's working fine.

![newsletter-error](https://cloud.githubusercontent.com/assets/20528998/21875017/c6a2aa04-d8b3-11e6-8578-6f9b7851a4ee.png)

2. It's giving error In Newsletter Problems Report -> Error Code 0 & Error Text - Syntax Error

![newsletter](https://cloud.githubusercontent.com/assets/20528998/21791025/1b860376-d71b-11e6-8b13-a9c17d2aedb8.PNG)

Please give feedback @magento-admin  @magento-team 

### Expected result

1.  Is should send mail

### Actual result

1. It's not sending Email

**Comment by veloraven** [comment]
@shahankitb997 thank you for your report.
I was not able to reproduce this issue.
Could you provide more information?
Are there any logs?
Does it happen to all emails? I mean for any customer and any template? I tried with the almost empty template (unsubscribe link and just some text like 'test') and there were no problems.
Could you also provide information about your environment? Like OS, versions of PHP, MySQL and web-server.

**Comment by shahankitb997** [comment]
Hi @veloraven   @magento-admin   @magento-team 

PHP 7,  MySQL 5.6.28-0ubuntu0.14.04.1. 

Please see additional changes in question. Issue raises when you put Space in Sender Name.

For example:

- Sender Name: This is test  [Gives Syntax Error]
- Sender Name: This  [Works fine]

**magento\vendor\magento\module-newsletter\Model\Queue.php**

Go to `public function sendPerSubscriber($count = 20)`

Line Number: 238

Currently

```
 )->setFrom(
                ['name' => $this->getNewsletterSenderEmail(), 'email' => $this->getNewsletterSenderName()]

```
It should be				
				
```
)->setFrom(
               ['name' => $this->getNewsletterSenderName(), 'email' => $this->getNewsletterSenderEmail()]

```

**Comment by hostep** [comment]
There is a PR for this which was recently approved: https://github.com/magento/magento2/pull/7771

Also, these issues seem to be duplicates of this one: https://github.com/magento/magento2/issues/7055 & https://github.com/magento/magento2/issues/4421

**Comment by Ctucker9233** [comment]
@okorshenko What version of Magento is this fix going to be released in?

**Comment by hostep** [comment]
Looks like this was recently fixed by https://github.com/magento/magento2/pull/6354

**Comment by okorshenko** [comment]
@Ctucker9233 this fix was delivered to develop branch. So, it will be released in Magento 2.2,
but if you need this fixed earlier for Magento 2.1, you can create a Pull Request to 2.1-develop branch and it will be released in the nearest 2.1 patch releases.

**Comment by veloraven** [comment]
According to [contributor guide](http://devdocs.magento.com/guides/v2.1/contributor-guide/contributing.html), tickets without response for two weeks should be closed.
If this issue still reproducible please feel free to create the new one: format new issue according to the [Issue reporting guidelines](https://github.com/magento/magento2/wiki/Issue-reporting-guidelines): with steps to reproduce, actual result and expected result and specify Magento version.
